src/actions/cloud-compute/cloud-compute.ts

```diff
@@ -42,190 +21,75 @@ export const runMeticulousTestsCloudComputeAction = async (): Promise<void> => {
-    );
-    return;
-  }
 
   // Compute the HEAD commit SHA to use when creating a test run.
   // In a PR workflow this will by default be process.env.GITHUB_SHA (the temporary merge commit) or
   // sometimes the head commit of the PR.
   // Users can also explicitly provide the head commit SHA to use as input. This is useful when the action is not
   // run with the code checked out.
   // Our backend is responsible for computing the correct BASE commit to create the test run for.
-  const head = headShaFromInput || getHeadCommitShaFromRepo();
+  const headSha = headShaFromInput || getHeadCommitShaFromRepo();
 
-  // Compute the base commit SHA to compare to for the HEAD commit.
-  // This will usually be the merge base of the PR head and base commit. In some cases it can be an older main branch commit,
-  // for example when running in a monorepo setup.
-  const { baseCommitSha, baseTestRun } = await getCloudComputeBaseTestRun({
-    apiToken,
-    headCommitSha: head,
-  });
+  const skippedTargets = projectTargets.filter((target) => target.skip);
+  const projectTargetsToRun = projectTargets.filter((target) => !target.skip);
 
-  let shaToCompareAgainst: string | null = null;
-  if (baseTestRun != null) {
-    shaToCompareAgainst = baseCommitSha;
-    logger.info(
-      `Tests already exist for commit ${baseCommitSha} (${baseTestRun.id})`
-    );
-  } else {
-    // We compute and use the base SHA from the code change event rather than the `baseCommitSha` computed above
-    // as `tryTriggerTestsWorkflowOnBase` can only trigger workflow for the HEAD `main` branch commit.
-    // `baseCommitSha` can be an older commit in a monorepo setup (in cases where we selectively run tests for a specific package).
-    // In such cases we won't be able to trigger a workflow for the base commit SHA provided by the backend.
-    // We will instead trigger test run for a newer base which is the base commit SHA from the code change event and
-    // will use that as the base to compare against. This is safe as `codeChangeBase` is guaranteed to be the same
-    // or newer commit to `baseCommitSha`.
-    const { base: codeChangeBase } = await getBaseAndHeadCommitShas(event, {
-      useDeploymentUrl: false,
-    });
-    if (codeChangeBase) {
-      const { baseTestRunExists } = await tryTriggerTestsWorkflowOnBase({
-        logger,
-        event,
-        base: codeChangeBase,
-        context,
-        octokit,
-      });
-
-      if (baseTestRunExists) {
-        shaToCompareAgainst = codeChangeBase;
-      }
-    }
-  }
+  // Single test run execution is a special case where we run a single test run with the "default" name.
+  // This will be the case when the user provides `app-url` and `api-token` inputs directly.
+  // This is used to simplify some of the logging and error handling.
+  const isSingleTestRunExecution =
+    projectTargetsToRun.length === 1 &&
+    projectTargetsToRun[0].name === "default";
```

Comment:
Hmm, I think we should use `projectTargets` (before the filtering is applied) here. If we have more than one project in the config, I think it's nice to be explicit in the logging about which one the logs are coming from even if a particular run only executed one.